Intersect.Server/Entities/ProjectileSpawn.cs

```diff
@@ -66,80 +66,74 @@ public bool IsAtLocation(Guid mapId, int x, int y, int z)
             return MapId == mapId && X == x && Y == y && Z == z;
         }
 
-        public bool HitEntity(Entity en)
+        public bool HitEntity(Entity targetEntity)
         {
-            var targetEntity = en;
-            if (targetEntity is EventPageInstance) return false;
-
-            var scalingStat = Enums.Stats.StatCount;
-
-            if (Parent.Spell != null && Parent.Spell.Combat != null)
+            Player targetPlayer = null;
+            if (targetEntity is Player)
```

Comment:
`if (targetEntity is Player targetPlayer)` if you can add more logic into the body of the if statement, or `var targetPlayer = targetEntity as Player;` otherwise. 

`(Player) targetEntity` I think will throw a cast exception, but `targetEntity as Player` will just result in null if it is not a player.